config/samples/default.yaml

```diff
@@ -8,4 +8,24 @@ metadata:
   name: default
   namespace: kyma-system
 spec:
-  # TODO(user): Add fields here
+  labels: {}
+  annotations: {}
+  companionConfig:
+    namespace: ai-core
+    configMapNames:
```

Comment:
are we expecting multiple configMaps?

Reply:
Although at the moment we have only one single ConfigMap, I want to provide an option to use multiple ones. This helps us to be flexible for the future.